docs/todo/FUNCTIONAL_MIGRATION_PLAN.md

```
@@ -0,0 +1,990 @@
+2. **Partial migration (some files done):**
+   ```bash
+   # Revert specific file
+   git checkout HEAD~1 -- src/modules/breakout/services/distribution.ts
+   
+   # Or reset to specific commit
+   git reset --hard <commit-hash>
+   ```
+
+3. **Adapter pattern (temporary bridge):**
+   If you need to support both patterns during transition:
+   
```

markdownlint-cli2 (0.20.0)

[warning] Trailing spaces
Expected: 0 or 2; Actual: 3

(MD009, no-trailing-spaces)